views/gitHub-loading.hbs

```diff
@@ -0,0 +1,51 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="ap-local-base-url" content="{{localBaseUrl}}" />
+    <title>{{title}}</title>
+    <link rel="stylesheet" href="/public/css/loading-screen.css" media="all" />
+    <link rel="stylesheet" href="/public/css/ghe-loading.css" media="all" />
```

Comment:
I don't think we need to create a separate file for one class. Makes more sense just to add it to the loading-screen file and treat that as the stylesheet for both cloud and server loading screens. Maybe just add a comment above the ghe specific styling.

Reply:
Separate CSS file removed, added the styling in `loading-screen.css`